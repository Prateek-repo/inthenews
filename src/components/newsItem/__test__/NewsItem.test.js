import {shallow, configure} from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import Container from '../NewsItem';


let wrapper;
configure({ adapter: new Adapter() });


const defaultProps = {
    title: "top headlines"
}

beforeEach( () => {
wrapper =shallow(
    <Container {...defaultProps}/>
  )
})

afterAll( () => {
    jest.clearAllMocks();
})

describe('<NewsItem/>', () => {

  it('SnapShot creation', () => {

    expect(toJson(wrapper)).toMatchSnapshot();

  })
})