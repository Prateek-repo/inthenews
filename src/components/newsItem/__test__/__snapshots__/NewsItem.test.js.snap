// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewsItem/> SnapShot creation 1`] = `
<div
  className="my-3"
>
  <div
    className="card"
  >
    <div
      style={
        Object {
          "display": "flex",
          "justifyContent": "flex-end",
          "position": "absolute",
          "right": "0",
        }
      }
    >
      <span
        className="badge rounded-pill bg-danger"
      />
    </div>
    <img
      alt="..."
      className="card-img-top"
      height="200"
      width="200"
    />
    <div
      className="card-body"
      style={
        Object {
          "backgroundColor": null,
        }
      }
    >
      <h5
        className="card-title"
        style={
          Object {
            "color": null,
          }
        }
      >
        top headlines
      </h5>
      <p
        className="card-text"
        style={
          Object {
            "color": null,
          }
        }
      />
      <a
        className="btn btn-sm btn-outline-dark"
        rel="noreferrer"
        target="_blank"
      >
        Read More...
      </a>
    </div>
  </div>
</div>
`;
